runtime/fieldmask.go

```diff
@@ -92,6 +105,47 @@ func FieldMaskFromRequestBody(r io.Reader, msg proto.Message) (*field_mask.Field
 	return fm, nil
 }
 
+func isDynamicProtoMessage(md protoreflect.MessageDescriptor) bool {
+	return md != nil && (md.FullName() == (&structpb.Struct{}).ProtoReflect().Descriptor().FullName() ||
+		md.FullName() == (&structpb.Value{}).ProtoReflect().Descriptor().FullName())
```

Comment:
I appreciate that this is effectively a type safe comparison, but I think it should be ok to just use the string paths for something as stable as this, which removes the import from this file
```suggestion
	return md != nil && (md.FullName() == "google.protobuf.Struct" ||
		md.FullName() == "google.protobuf.Value"
```

See https://github.com/grpc-ecosystem/grpc-gateway/blob/v2/runtime/query.go#L251 for a precedent for this.

Reply:
Thanks.  Updated this as requested.